azurelinuxagent/common/cgroupapi.py

```diff
@@ -54,9 +57,12 @@ def cgroups_supported():
         distro_name = distro_info[0]
         try:
             distro_version = FlexibleVersion(distro_info[1])
+            major_minor_version = "{0}.{1}".format(distro_version.major, distro_version.minor)
         except ValueError:
             return False
-        return distro_name.lower() == 'ubuntu' and distro_version.major >= 16
+        return ((distro_name.lower() == 'ubuntu' and distro_version.major >= 16) or
+                (distro_name.lower() in ("centos", "redhat") and distro_version.major >= 7 and
+                 major_minor_version != "7.1"))
```

Comment:
we stated (in setup.py) there is a systemctl bug on 7.1 and also, I don't find 7.1 image form marketplace for testing. So I marked 7.1 not to enable.

Reply:
what is the issue with 7.1?

Reply:
I don't know the issue. I'm going by how setup.py was implemented. This was the comment stated.
```
            if version.startswith("7.1"):
                # TODO this is a mitigation to systemctl bug on 7.1
                set_sysv_files(data_files)
```

Reply:
can you investigate? what happens if you just enable it?

Reply:
As I said, I don't find image from marketplace to test it.

Reply:
Maybe I can try if DCR can create the VM

Reply:
what are the minimum versions tested in DCR and endorsed? maybe we should limit to those

Reply:
Minimum version tested `7.8`

Reply:
how about endorsed?

Reply:
https://docs.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros

supported versions `6.x, 7.x, 8.x`

Reply:
ok, then let's add to DCR the minimum endorsed version that we can test and then enable the feature only on that and later versions (i.e. we should not enable this feature unless it is tested)

Reply:
updated